### ISSUE TITLE
Feature request: send all instant forum notifications (not just one until that one is confirmed)

### ISSUE THREAD

**Issue by bersbersbers** [WoltLab/WCF#3109]
I am looking for a way to have an email sent for each new post in a thread, not just one and then no more until the first one has been "confirmed" in some way. I have learned from the code that "new post" notifications are stackable, but I see no way to configure this. Similarly, for stackable notifications, an email is only sent when a notification is only sent if the event is "new". So is there no way to do this currently? I would be OK with some code modifications myself, but they should apply only to this exact kind of notification (email about new post in thread).

**Comment by dtdesign** [comment]
We do not support this kind of behavior at this time, especially considering the severe impact this would have on mail traffic. The suggested behavior is only useful for forums that don't get visited by their users on a regular basis, for example, internal sites.

The current behavior is not only by design, but also reflected by the actual implementation, making changes quite difficult to implement without unwanted side effects. Adding such a toggle to the admin panel is not an option, because this is pretty much a self destruct button for the average hosting plan in regards to the email server reputation. Once the damage has been done, there is almost no way to recover from this and realistically there is no amount of warning messages that would prevent users from "accidentially" toggling it on!

Long story short: This edge case is not and will not be officially supported. If you need something in this direction and are very much aware of the consequences, please feel free to contact us, we can provide you with a solution.
